league.js

```diff
@@ -0,0 +1,693 @@
+                    var captain = null;
+                    function processPlayer(name, rating) {
+                        name = name.value;
+                        rating = rating.value;
+                        var player = {
+                            name: self.canonicalUsername(name),
+                            rating: rating,
+                            team: team
+                        };
+                        if (name != player['name'] && name[name.length-1] == '*') {
+                            captain = player;
+                        }
+                        return player;
+                    }
+                    roster.push(processPlayer(row['board 1'], row['rating 1']));
+                    roster.push(processPlayer(row['board 2'], row['rating 2']));
+                    roster.push(processPlayer(row['board 3'], row['rating 3']));
+                    roster.push(processPlayer(row['board 4'], row['rating 4']));
+                    roster.push(processPlayer(row['board 5'], row['rating 5']));
+                    roster.push(processPlayer(row['board 6'], row['rating 6']));
+
+                    team['captain'] = captain;
+                    team['roster'] = roster;
+
+                    newTeams.push(team);
+                });
+                self._teams = newTeams;
+                _.each(self._teams, function(team) {
+                    _.each(team.roster, function(player) {
+                        if (player && player.name) {
+                            if (firstRosterUpdate && player.rating) {
+                                lichess.setPlayerRating(player.name, player.rating);
+                            }
+                            lichess.getPlayerRating(player.name, true).then(function(rating) {
+                                player.rating = rating;
+                                self._playerRatingsLastUpdates[player.name] = moment.utc();
+                            }, function(error) {
+                                console.error("Error updating rating: " + JSON.stringify(error));
+                            });
+                        }
+                    });
+                });
+
+                callback(undefined, self._teams);
+            }
+        );
+    },
+
+    //--------------------------------------------------------------------------
+    // Figures out the current scheduling information for the round.
+    //--------------------------------------------------------------------------
+    'refreshCurrentRoundSchedules': function(callback) {
+        var query_options = {
+            'min-row': 1,
+            'max-row': 100,
+            'min-col': 1,
+            'max-col': 8,
+            'return-empty': true
+        }
+        var self = this;
+        spreadsheets.getPairingRows(
+            self.options.spreadsheet,
+            query_options,
+            function(err, rows) {
+                if (err) { return callback(err, rows); }
```

Comment:
this call back takes rows and the callback below takes pairings.

should this argument be left off?